module/mempool/herocache/backdata/heropool/pool.go

```diff
@@ -169,20 +168,20 @@ func (p Pool) Head() (flow.Entity, bool) {
 func (p *Pool) sliceIndexForEntity() (i EIndex, hasAvailableSlot bool, ejectedEntity flow.Entity) {
```

Comment:
With changes in this method, please indicate the returned slice index is not claimed from the `free` state, i.e., a switch state is needed by the caller.

Reply:
understood, I made a modification in the doc, pls take look